gradle-plugin/src/main/kotlin/org/mozilla/android/MozillaPlugin.kt

```diff
@@ -39,6 +48,170 @@ class MozillaPlugin : Plugin<Project> {
         project.extensions.extraProperties.set("localProperties", localProperties)
     }
 
+    private fun substituteLocalGeckoView(project: Project) {
```

Comment:
nit: this is a long function. It'd be nice for it to be in its own file with private helper functions.